Streetcode/Streetcode.BLL/MediatR/Team/GetAllMain/GetAllMainTeamHandler.cs

```diff
@@ -26,10 +29,14 @@ public async Task<Result<IEnumerable<TeamMemberDTO>>> Handle(GetAllMainTeamQuery
 
             if (team is null)
             {
-                return Result.Fail(new Error($"Cannot find any team"));
+                const string errorMsg = $"Cannot find any team";
```

Comment:
why somewhere you have const and somewhere not?

Reply:
There are constant messages like this one: "Cannot find any team".
And dynamicaly changed messages like this: $"GetAllSubtitlesQuery handled successfully. Retrieved {subtitleDtos.Count()} subtitles" that can not be constants.